File: Software Development Module/Week12/PhoneNumber.java
/*
PhoneNumber.java
Billy Cussen
23/07/2020
*/

public class PhoneNumber{

    //Data Members
    private String input;
    private String output;

    //Constructor and Setter
    public PhoneNumber(String input){
        this.input = input;
        this.output = new String();
    }

    //Compute & Getter
    public String computeAndGetResult(){

        if((this.input.substring(0,2).equals("08"))){
            if((this.input.charAt(2)=='3'||this.input.charAt(2)=='5'||this.input.charAt(2)=='6'||this.input.charAt(2)=='7'||this.input.charAt(2)=='9')){
                this.output = "This is a valid Phone Number";
            } else{
                this.output = "This is not a valid Phone Number";
            }
        } else{
            this.output = "This is not a valid Phone Number";
        } 

        return this.output;
    }
}

File: Python/Software Development Module - Python/Week12/PhoneNumber.py
"""
PhoneNumber.py
Billy Cussen
15/10/2020
"""

class myPhoneNumber:
    def __init__(self, phoneNumber):
        self.phoneNumber = phoneNumber

    def computePhoneNumber(self):
        if(self.phoneNumber[0:2] == "08" and 
        (self.phoneNumber.find("3")==2 or self.phoneNumber.find("5")==2 or self.phoneNumber.find("6")==2 
        or self.phoneNumber.find("7")==2 or self.phoneNumber.find("9")==2)):
            print("Your phone number is valid")
        else:
            print("Your phone number is invalid")